L1Trigger/L1TTwinMux/interface/L1MuTMChambPhContainer.h

```diff
@@ -0,0 +1,67 @@
+//-------------------------------------------------
```

Comment:
On second thought: It this product going to be stored in an event at some point? If not then it might stay here.